tests/test_diff.py

```diff
@@ -1,54 +1,52 @@
 # Copyright (c) 2019 The Regents of the University of Michigan
 # All rights reserved.
 # This software is licensed under the BSD 3-Clause License.
-import unittest
+import pytest
 import signac
 from tempfile import TemporaryDirectory
 
 
-class BaseDiffTest(unittest.TestCase):
+class TestDiffBase():
 
     project_class = signac.Project
 
+    @pytest.fixture(autouse=True)
     def setUp(self):
         self._tmp_dir = TemporaryDirectory(prefix='signac_')
-        self.addCleanup(self._tmp_dir.cleanup)
+        # self.addCleanup(self._tmp_dir.cleanup)
```

Comment:
Is this function no longer needed? If yes, should the line be removed instead of commented out?